lib/blaze-wooless-shortcodes.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+add_shortcode('bc_current_year', 'blaze_wooless_current_year');
```

Comment:
if we can use [bc_current_time format="d/m/Y"] and it will display 06/06/24, it's better